src/scss/components/eyebrow/index.scss

```diff
@@ -0,0 +1,126 @@
+// ----- BASE ELEMENTS ---------------
+
+.eyebrow {
+  overflow: hidden;
+  width: 100%;
+  height: 56px;
+  background-color: #000000;
+}
+
+.img-left {
+  float: left;
+  width: 159px;
+  height: 56px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.right-menu {
+  float: right;
+}
+
+.eyebrow a img {
+  width: 119px;
+  height: 24px;
+}
+
+.eyebrow a {
+  float: left;
+  font-size: 13px;
+  color: #ffffff;
+  text-align: center;
+  padding: 1rem 9px;
+  text-decoration: none;
+  line-height: 1;
+  height: 56px;
+}
+
+.eyebrow p {
+  color: #ffffff;
+  line-height: 1.5;
+}
+
+.dropdown {
+  float: right;
+  overflow: hidden;
+}
+
+.dropdown .dropbtn {
```

Comment:
### We are missing multiple `dropdown` states, including `disabled`, `selected`, `disabled hover`,... :

<img width="1123" alt="Screen Shot 2020-07-27 at 5 47 40 PM" src="https://user-images.githubusercontent.com/10120600/88599453-6b46d300-d031-11ea-8763-27b83bf23f6c.png">

Reply:
Changes have been done, Please review